services/blockchain-connector/shared/sdk/reward.js

```diff
@@ -36,6 +36,19 @@ const getRewardTokenID = async () => {
 	return rewardTokenID;
 };
 
+const getDefaultRewardAtHeight = async height => {
+	try {
+		const defaultReward = await invokeEndpoint('reward_getDefaultRewardAtHeight', { height });
+		return defaultReward.error ? null : defaultReward;
+	} catch (err) {
+		if (err.message.includes(timeoutMessage)) {
+			throw new TimeoutException(`Request timed out when calling 'getDefaultRewardAtHeight' for block height:${height}`);
+		}
```

Comment:
Please return the response as is from the connector. Let us also start logging the errors like suggested below. Please import and initialize the logger at the top of the file.

```
		return defaultReward;
	} catch (err) {
		if (err.message.includes(timeoutMessage)) {
			throw new TimeoutException(`Request timed out when calling 'getDefaultRewardAtHeight' for block height: ${height}`);
		}
		logger.warn(`Error returned when invoking 'reward_getDefaultRewardAtHeight' with param: ${height}.\n${err.stack}`);
```

Reply:
Applied.